common.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
```

Comment:
Need to use env

Reply:
changed for all the scripts in this repo